src/terra-dev-site/IssueForm/PackageSelect.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Field from 'terra-form-field';
+import Select from 'terra-form-select';
+import { getPackages, propTypes } from './Helper';
+
+const packageList = getPackages().map(currentPackage => (
+  <Select.Option
+    value={currentPackage}
+    display={currentPackage}
+    key={currentPackage}
+  />
+));
+
+const PackageSelect = (props) => {
+  const {
+    setPackage,
+  } = props;
+
+  return (
+    <Field
+      label="Package Name"
+      help="Selected package for issue / feature request."
+    >
+      <Select
+        aria-label="Drop down menu for selecting the package related to the bug report or feature request."
+        onChange={setPackage}
+        defaultValue="terra-action-footer"
```

Comment:
Should we also support logging issues on a per repository basis? I can see someone trying to log an issue to `terra-toolkit` or `generator-terra-module`. If that's the case, we may have to rethink how we separate + populate the fields

Reply:
At this point, I only have core, framework, and clinical in there. The repository is set in the background based on the package selected from the list (it does get displayed in the preview, for informational purposes). @emilyrohrbough also suggested adding other repositories; however, I agree that it might require some restructuring, since that package select will get really crowded.

@bjankord @yuderekyu Should we make the repository selection visible and populate the package list based on that, or add the assorted repository packages to the existing list and preserve the current functionality?

Reply:
could the searchable form select be used here instead so it filters on the component name people start to type?

Reply:
I discussed this with @bjankord, and he suggested adding repos beyond core / framework / clinical as enhancements at a later time.